pgmpy/factors/CPD.py

```diff
@@ -11,6 +12,7 @@
 from pgmpy.extern import tabulate
 from pgmpy.extern import six
 from pgmpy.extern.six.moves import range, zip
+from warnings import warn
```

Comment:
@anaviltripathi Please refer to PEP8 (https://www.python.org/dev/peps/pep-0008/#imports) for how to group import statements. `warnings` is a standard library import so should be in the top group.